appinventor/components/src/com/google/appinventor/components/runtime/Direction.java

```diff
@@ -0,0 +1,25 @@
+
```

Comment:
Don't forget to include emacs file variables, MIT copyright, and Apache license info.